app/Core/Repository/ServiceRepository.cs

```diff
@@ -22,45 +21,69 @@ public DataTable SearchService(string searchValue)
             UpgradeFile upgradeFile = new UpgradeFile();
             return upgradeFile.Load(query, parameters);
         }
-        public bool SaveService(Services services)
+        public bool SaveService(Services service)
         {
             string sql;
+            Dictionary<string, string> parameters;
 
-            bool saveState = services.Id > 0 ? true : false;
+            // Determine if we're updating an existing service or inserting a new one
+            bool saveState = service.Id > 0;
 
             if (saveState)
             {
-                sql = "UPDATE services SET id=@Id WHERE id=@Id;";
+                // Update existing service
```

Comment:
ServiceCode still not yet reflecting after updating.

**Before:**
![image](https://github.com/user-attachments/assets/381e8f22-d07f-4f32-9717-a8496217c326)

**After:**
![image](https://github.com/user-attachments/assets/bd902fe2-da2f-4d9c-ad2e-b6cab4a659c1)

RESULT:
![image](https://github.com/user-attachments/assets/75e05e6e-7900-4d1b-acea-dbac972fb462)